courses/cunix/ex07/src/linked_list.c

```diff
@@ -0,0 +1,120 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include "linked_list.h"
+
+node_t *list_create (void *data)
```

Comment:
unfortunately I couldn't run tests for ex07:
![image](https://user-images.githubusercontent.com/39953699/132191911-67085eed-7e10-4793-b811-f792f5696c24.png)